scripts/tooling/blackhole_recovery/data/bh_flm/eeprom.c

```diff
@@ -25,6 +25,7 @@ struct spi_nor_chip {
 };
 
 #define SPI_NOR_WIP_BIT 0x01 /* Write In Progress bit in status register */
+#define FLASH_PAGE_SIZE 0x100      /* Flash page size in bytes (256 bytes) */
```

Comment:
nit: Like Chris said `SPI_NOR_PAGE_SIZE` is probably better to not confuse this with internal flash